beacon_node/lighthouse_network/src/peer_manager/mod.rs

```diff
@@ -943,6 +945,13 @@ impl<E: EthSpec> PeerManager<E> {
         }
     }
 
+    fn maintain_trusted_peers(&mut self) {
+        let trusted_peers = self.trusted_peers.clone();
+        for trusted_peer in &trusted_peers {
+            self.dial_peer(trusted_peer.clone());
+        }
```

Comment:
Can get rid of this one, but it's likely not really important

```suggestion
        for trusted_peer in trusted_peers {
            self.dial_peer(trusted_peer);
        }
```